java-security/src/main/java/com/sap/cloud/security/x509/X509CertificateExtractor.java

```diff
@@ -39,17 +37,10 @@ public static X509CertificateExtractor getInstance() {
 	 * @return the client certificate object
 	 */
 	@Nullable
-	public X509Certificate getClientCertificate(HttpServletRequest request) {
+	public String getClientCertificate(HttpServletRequest request) {
```

Comment:
this somewhat makes the X509CertificateExtractor kind of obsolete, maybe you can put it into the AbstractAutheniticator implementation as it was used in both Authenticators